ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js

```diff
@@ -158,6 +162,25 @@ module.exports = plugin(
       )
       this.settingsSrc.currentView.turnOffEditing()
     },
+    setupSpellcheck: function() {
+      if(!properties.isSpellcheckEnabled()){
+        return
+      }
+      const spellcheckView = Marionette.ItemView.extend({
+        template: () => <RadioComponent value={this.model.get('spellcheck')} label="Spellcheck" options={[{
+          label: 'Yes',
+          value: true
+        }, {
+          label: 'No',
+          value: false
+        }]}
+          onChange={(value) => {
+            this.model.set('spellcheck', value)
```

Comment:
Should `change:spellcheck` be added here? https://github.com/codice/ddf/pull/4698/files#diff-1d3ab149b2771a1eaea93cb9f46d7c0bR63